libs/books/data-access/src/lib/+state/reading-list.selectors.ts

```diff
@@ -17,7 +17,7 @@ export const getReadingListState = createFeatureSelector<
 const {
   selectEntities,
   selectAll,
-  selectTotal
+  selectTotal,
```

Comment:
Remove extra , from her and check for all files where you have made such mistake.